roomserver/internal/query/query.go

```diff
@@ -242,6 +242,24 @@ func (r *Queryer) QueryMembershipsForRoom(
 		return err
 	}
 
+	if request.Sender == "" {
```

Comment:
Please comment what the behaviour is supposed to be with an absent sender.

Reply:
Done, although I had already added a comment into the `api` package.